packages/tc-ui/src/primitives/multiple-choice/server.jsx

```diff
@@ -13,6 +14,42 @@ const localOnChange = (event, currentValues, onChange) => {
 	onChange(propertyName, parentCode, values);
 };
 
+const getClass = isNested =>
+	isNested ? 'treecreeper-relationship-annotate__tooltip' : '';
+
+const getDescription = (checkboxValue, type, parentCode, isNested) => {
+	const enumWithMeta = getEnums({ withMeta: true })[type];
+	const optionDefs = Object.values(enumWithMeta.options);
+	const hasOptionDescriptions = !!optionDefs[0].description;
+	if (!hasOptionDescriptions) {
+		return null;
+	}
+	const [{ description }] = optionDefs.filter(
+		({ value }) => value === checkboxValue,
+	);
+	return (
+		<div
+			className="tooltip-container"
+			id={`tooltip-target-${parentCode}-${checkboxValue}`}
+		>
+			<span id={`tooltip-target-${parentCode}-${checkboxValue}`} />
+			<i
+				aria-label={`help for ${checkboxValue}`}
+				className="o-icons-icon o-icons-icon--info treecreeper-help-icon"
+			/>
+			<div
+				className={getClass(isNested)}
```

Comment:
I think we should use this when not nested too (we don't have any examples elsewhere in Biz Ops, but I think it's a nice feature)

Reply:
...but I see in the css that implementing that would be extra work, so it's fine to leave it as it is for now

Reply:
@wheresrhys I have modified it to address your suggestion

Reply:
🙌